client/src/Components/Profile/AddItem.js

```diff
@@ -0,0 +1,276 @@
+import React, { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import { makeStyles } from '@mui/styles';
+import { styled } from '@mui/material/styles';
+// import FormControlLabel from '@mui/material/FormControlLabel';
+// import FormControl from '@mui/material/FormControl';
+import TextField from '@mui/material/TextField';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import axios from 'axios';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    flexGrow: 1,
+  },
+  bold: {
+    fontWeight: 800,
+  },
+  hover2: {
+    color: '#ffffff',
+    '&:hover': {
+      color: '#f0CC71',
+    },
+  },
+  input: {
+    color: '#FFFFFF',
+  },
+  formLabel: {
+    color: '#FFFFFF',
+  },
+  menu: {
+    '& .MuiPaper-root': {
+      backgroundColor: '#2C2C2C',
+    },
+  },
+}));
+
+const AddItem = () => {
+  const classes = useStyles();
+
+  const Input = styled('input')({
+    display: 'none',
+  });
+
+  const [newItem, setNewItem] = useState({
+    itemCategory: 'Select Category',
+    itemName: '',
+    itemDesc: '',
+    itemZIP: '',
+  });
+
+  const [itemFormOpen, setItemFormOpen] = useState(false);
+
+  const noItemPic =
+    'https://cdn4.iconfinder.com/data/icons/ionicons/512/icon-camera-512.png';
+
+  const handleClick = () => {
+    setItemFormOpen(true);
+  };
+
+  const handleClose = () => {
+    setItemFormOpen(false);
+  };
+
+  const handleChange = (event) => {
+    setNewItem({
+      ...newItem,
+      [event.target.name]: event.target.value,
+    });
+    // console.log(newItem);
+  };
+
+  const handleSubmit = () => {
+    console.log('Sending new item data: ', newItem);
+    axios
+      .post(`/items`, newItem)
+      .then((postResponse) => {
+        console.log('Received post response:');
+        console.log(postResponse);
+      })
+      .catch((err) => {
+        console.log('Error received from post request:');
+        console.log(err);
+      });
+  };
+
+  return (
+    <div className={classes.root}>
```

Comment:
Try to avoid using div's. Use a React Fragment or Grid container, and in this case you could delete the div and use the next Grid container as the main surrounding element.